card/src/main/java/com/adyen/checkout/card/CardConfiguration.java

```diff
@@ -32,6 +32,11 @@
  */
 public class CardConfiguration extends Configuration {
 
+    public enum AddressVisibility {
```

Comment:
Maybe move this to it's own file like `KCPAuthVisibility`? I think it could be in the base module since other components might have Address inputs as well.

Reply:
done